wpilibc/src/main/native/include/frc/DoubleSolenoid.h

```diff
@@ -88,6 +88,39 @@ class DoubleSolenoid : public wpi::Sendable,
    */
   void Toggle();
 
+  /**
+   * Returns true if the double solenoid is in a forward state.
+   * @return true if the double solenoid is in a forward state.
+   */
+  virtual bool IsForward() const;
+
+  /**
+   * Returns true if the double solenoid is in a reverse state.
+   * @return true if the double solenoid is in a reverse state.
+   */
+  virtual bool IsReverse() const;
+
+  /**
+   * Returns true if the double solenoid is in an off state.
+   * @return true if the double solenoid is in an off state.
+   */
+  virtual bool IsOff() const;
+
+  /**
+   * Sets the double solenoid to a forward state
+   */
+  virtual void Forward();
```

Comment:
For consistency/clarity these should be `SetForward`, `SetReverse` etc.

Reply:
should on/off be SetOn, Set Off?